src/status_im/contexts/keycard/nfc/effects.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.contexts.keycard.nfc.effects
+  (:require [keycard.keycard :as keycard]
+            [utils.re-frame :as rf]))
+
+(rf/reg-fx :effects.keycard/check-nfc-enabled
+ (fn []
+   (keycard/check-nfc-enabled
+    {:on-success #(rf/dispatch [:keycard/on-check-nfc-enabled-success %])})))
+
+(rf/reg-fx
+ :effects.keycard.ios/start-nfc
+ (fn [args]
+   (keycard/start-nfc args)))
+
+(rf/reg-fx
+ :effects.keycard.ios/stop-nfc
+ (fn [args]
+   (keycard/stop-nfc args)))
```

Comment:
Small comment: could we pass the reference to the functions directly? Like this:
```clojure
(rf/reg-fx :effects.keycard.ios/start-nfc keycard/start-nfc)
(rf/reg-fx :effects.keycard.ios/stop-nfc keycard/stop-nfc)
```